pyparsing/core.py

```diff
@@ -243,7 +245,8 @@ def _should_enable_warnings(
 alphanums: str = alphas + nums
 printables: str = "".join([c for c in string.printable if c not in string.whitespace])
 
-
+class ParserElement:
```

Comment:
Needs explanation here

Reply:
I've upgraded cython version to 3.0.11, and now its working. So removed that.